src/components/ListingDetails/ListingDetails.tsx

```diff
@@ -1,14 +1,14 @@
 import { Card, Avatar, Divider } from 'antd';
 import { HiOutlineLocationMarker } from 'react-icons/hi';
-import { useAppSelector } from '../../store/hooks';
+import { useTypedSelector } from '../../store/useTypedSelector';
 import { IListingDetailsProps } from './listingDetails.types';
 
 export const ListingDetails: React.FC<IListingDetailsProps> = ({
   data,
   id,
 }) => {
-  const { authData } = useAppSelector((state) => state.auth);
-
+  const { authData } = useTypedSelector((state) => state.auth);
+  console.log(authData);
```

Comment:
do we need that

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/37/commits/9c3bff5628ec16686f9f6220ae73995d9479c4da